modules/deployment-impl/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>mule-modules</artifactId>
+        <groupId>org.mule.modules</groupId>
+        <version>4.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>mule-module-deployment-impl</artifactId>
+    <packaging>jar</packaging>
+    <name>Mule Deployment Model Implementation Module</name>
```

Comment:
Is this the implementation of deployment model or deployment?. If it were the implementation of deployment model I would expect the artifact id to be org.mule.modules:mule-module-deployment-model-impl instead of org.mule.modules:mule-module-deployment-impl